packages/browser-wallet/src/background/window-management.ts

```diff
@@ -0,0 +1,71 @@
+import { ExtensionMessageHandler, InternalMessageType } from '@concordium/browser-wallet-message-hub';
+
+import { height, width } from '@popup/constants/dimensions';
+import { spawnedPopupUrl } from '@shared/constants/url';
+// eslint-disable-next-line import/no-cycle
+import bgMessageHandler, { onMessage } from './message-handler';
```

Comment:
Hmm, could we restructure/add a third file, so we can avoid this?

Reply:
I'll see what I can do! ⛑️